ConnectorAutocadCivil/ConnectorAutocadCivil/UI/ConnectorBindingsAutocadCivil.Receive.cs

```diff
@@ -519,23 +517,23 @@ void FlattenConvertedObject(object item)
 
             // add property sets if this is Civil3D
 #if CIVIL2021 || CIVIL2022 || CIVIL2023 || CIVIL2024
```

Comment:
I know this is pre-existing, but maybe worth doing the greater than approach?

Reply:
This will be done in a separate ticket to improve our preprocessor directives accross all sharp connectors